### PR TITLE
[Enhancement] Slider Thumb Size and Convert Range

### PR DESCRIPTION
**Pull request by dimonovdd**
**What do you think about it?**

### CODE REVIEW

**Comment by dimonovdd** [comment]
We also need to make the position of the thumb to be set through its center, not its left edge

**Comment by jsuarezruiz** [comment]
Oh, here we have the first external contribution. Thanks!.

Mnn, what do you think if we simplify this option and have only one `ThumbSize` property?

To use it, we would use  `ThumbSize = "50"` or `ThumbSize = "24, 48"`.

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]
Oh, here we have the first external contribution. Thanks!.

Mnn, what do you think if we simplify this option and have only one `ThumbSize` property?

To use it, we would use  `ThumbSize = "50"` or `ThumbSize = "24, 48"`.

About the CornerRadius option. Mnn, it makes sense because we are using a Frame to draw the Thumb, but if you need a specific shape, must use a modified ControlTemplate. What do you think if we leave this option out, and whoever wants to use it, just have to modify the ControlTemplate?

**Comment by dimonovdd** [comment]
thaks, maybe this is a way?
```xml
    <controls:Slider
                TrackSize="4"
                Minimum="0"
                MinimumTrackColor="LightGreen"
                Maximum="10"
                MaximumTrackColor="LightGreen"
                Value="2">
        <controls:Slider.Thumb>
            <Frame BackgroundColor="Black" CornerRadius="10" HeightRequest="40" WidthRequest="40" Padding="0" Margin="0">
                <Image Margin="5" VerticalOptions="FillAndExpand" HorizontalOptions="FillAndExpand">
                    <Image.Source>
                        <iconsB:IconSource Icon="GithubAlt" Color="Red"/>
                    </Image.Source>
                </Image>
            </Frame>
        </controls:Slider.Thumb>
    </controls:Slider>
```

image

**Comment by dimonovdd** [comment]
image
image
image

File: src/TemplateUI/Controls/Slider/Slider.cs [hunk 1]
```diff
@@ -170,14 +161,20 @@ void OnThumbPanUpdated(object sender, PanUpdatedEventArgs e)
 
                     if (Device.RuntimePlatform == Device.Android)
                         totalX += _thumb.TranslationX;
-
-                    var position = totalX < 0 ? 0 : totalX > _trackBackground.Width - _thumb.Width ? _trackBackground.Width - _thumb.Width : totalX;
-                    Value = position * Maximum / _trackBackground.Width;
+
```
Comment (jsuarezruiz) [inline src/TemplateUI/Controls/Slider/Slider.cs]: I will review the calculations, when moving to the right there is at the beginning a little jump to the left. 
thumbsize

**Review by jsuarezruiz — CHANGES_REQUESTED** [review]

**Comment by dimonovdd** [comment]
@jsuarezruiz 
Thank you for watching this.
I fixed bugs on calculating slider value and thumb position.

**Review by jsuarezruiz — APPROVED** [review]

**Pull request merged**